javascript/hashTables/hashTable.test.js

```diff
@@ -0,0 +1,30 @@
+const  HashTable  = require('./hashTable');
+
+describe('Hash table implementation', () => {
+  test('can initialize a new hash table', () => {
+    let hashTable = new HashTable();
+  })
+  test('can set a key/value to hashtable results in the value being in the data structure', () => {
+    hashTable.set('Paloma', 3);
```

Comment:
Either add `let hashTable = new HashTable();` between lines 7 and 8. Or put on line two and rename line five to something like `hashtable1` or something. The reason is, as of right now, `hashTable` is only being used in the first test because it is only available within that code block, so the other test can't use it🙅‍♀️; either we add it to every test or make it global, where all the tests can use it